src/index.html

```diff
@@ -120,6 +130,10 @@ <h2>I want more!</h2>
       </div>
     </div>
 
+    <script
+    src="https://code.jquery.com/jquery-3.7.1.slim.min.js"
```

Comment:
do the icons require jquery?

Reply:
oh I see you're using some jquery now, it's not entirely necessary but it is more convenient than plain JS

maybe it'll be helpful if we start doing custom sorting/filtering

Reply:
No, it is used to simplify the theme transition. I'm open to avoiding jQuery, but it seems silly considering its ubiquity & the benefits when it comes to keeping custom scripts to a minimum.